src/internal_events/common.rs

```diff
@@ -25,6 +25,11 @@ impl InternalEvent for EndpointBytesReceived<'_> {
             "protocol" => self.protocol.to_owned(),
             "endpoint" => self.endpoint.to_owned(),
         );
+        histogram!(
+            "component_received_bytes", self.byte_size as f64,
+            "protocol" => self.protocol.to_owned(),
+            "endpoint" => self.endpoint.to_owned(),
```

Comment:
```suggestion
```

We've actually been trying to drop these extra tags since they inflate the cardinality of these metrics (see https://github.com/vectordotdev/vector/issues/15426). Are they tags that you need?

Reply:
done!